code/__DEFINES/misc.dm

```diff
@@ -499,6 +499,7 @@ GLOBAL_LIST_INIT(pda_styles, sort_list(list(MONO, VT, ORBITRON, SHARE)))
 
 #define VOMIT_TOXIC 1
 #define VOMIT_PURPLE 2
+#define VOMIT_NANITE 3 //SKYRAT EDIT ADD
```

Comment:
You should say in your SKYRAT EDIT comment the reason why you're making said edit, so it's easier to find in the future.

Reply:
```suggestion
#define VOMIT_NANITE 3 //SKYRAT EDIT ADD - Nanite Slurry
```